compiler/rustc_session/src/config.rs

```diff
@@ -1441,6 +1441,7 @@ impl CheckCfg {
         // These three are never set by rustc, but we set them anyway: they
         // should not trigger a lint because `cargo doc`, `cargo test`, and
         // `cargo miri run` (respectively) can set them.
+        ins!(sym::clippy, no_values);
```

Comment:
It needs to be added in [tests/ui/check-cfg/well-known-values.rs](https://github.com/rust-lang/rust/blob/4ae1e79876fcee94a56c1edf7c029a677084d7fc/tests/ui/check-cfg/well-known-values.rs).

*You will also probably need to bless the directory*